### PR TITLE
Create ssh-tunnel-manager-beta.rb

### PR DESCRIPTION
**Pull request by RJHsiao**
#### Adding a new cask

- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by RJHsiao** [comment]
They release their beta version on facebook page: https://www.facebook.com/sshtunnelmanager/

File: Casks/ssh-tunnel-manager-beta.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'ssh-tunnel-manager-beta' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://dl.devmate.com/org.tynsoe.sshtunnelmanager/beta/SSHTunnelManager.zip'
```
Comment (vitorgalvao) [inline Casks/ssh-tunnel-manager-beta.rb]: Needs a `# dl.devmate.com/org.tynsoe.sshtunnelmanager was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (RJHsiao) [inline Casks/ssh-tunnel-manager-beta.rb]: Hi @vitorgalvao 
Please check it out and let me know if I lost to do something necessary again.

Comment (vitorgalvao) [inline Casks/ssh-tunnel-manager-beta.rb]: Everything looks fine. Thank you.

**Pull request merged**